manifests/07-downloads-deployment.yaml

```diff
@@ -89,10 +89,14 @@ spec:
               os.mkdir(arch)
               for operating_system in ['linux', 'mac', 'windows']:
                   os.mkdir(os.path.join(arch, operating_system))
+                  target_path = os.path.join(arch, operating_system, 'LICENSE')
```

Comment:
You may want to add a new `TestLicense()` to 
https://github.com/openshift/console-operator/blob/master/test/e2e/downloads_test.go

Reply:
+1 on adding testes

Reply:
I added a new test `TestDonwloadsLicense()` and some tests for pkg.

The new commit have passed `make verify` and `make test-unit` on my local machine.

Reply:
Ran it on a cluster?